build.gradle

```diff
@@ -84,6 +84,10 @@ repositories {
     mavenCentral()
 }
 
+if (JavaVersion.current() <= JavaVersion.VERSION_1_8) {
+    sourceSets.test.java.exclude '**/AnnotationPosition*.java'
+}
```

Comment:
Why do we have to exclude this test here, if inside of it we have:

```java
        // Java 8 has some bugs in Language Model API, see https://bugs.openjdk.org/browse/JDK-8031744
        // Therefore, Jilt cannot correctly detect TYPE_USE-only annotations.
        Assume.assumeTrue(jreVersion >= 9);
```

?

Reply:
`.getAnnotatedOwnerType() // ! @since 9`